src/Stripe82Score.jl

```diff
@@ -293,7 +292,7 @@ function celeste_to_df(results::Vector{OptimizedSource})
 
         function get_median_fluxes(i::Int)
             ret = Array(Float64, 5)
-            ret[3] = exp(vs[ids.r1[i]] + 0.5 * vs[ids.r2[i]])
+            ret[3] = exp(vs[ids.r1[i]])
```

Comment:
I don't think this changes the scores substantively. It's just that I had changed the name of the function to indicate median but left the formula for the mean.